var/spack/repos/builtin/packages/damask-grid/package.py

```diff
@@ -17,9 +17,22 @@ class DamaskGrid(CMakePackage):
 
     depends_on('pkgconfig', type='build')
     depends_on('cmake@3.10:', type='build')
-    depends_on('petsc@3.14.0:3.14,3.15.1:3.15', when='@3.0.0-alpha4')
-    depends_on('petsc@3.14.0:3.14,3.15.1:3.16', when='@3.0.0-alpha5')
-    depends_on('hdf5+fortran')
+    depends_on('petsc+hdf5+mpi@3.14.0:3.14,3.15.1:3.15', when='@3.0.0-alpha4')
+    depends_on('petsc+hdf5+mpi@3.14.0:3.14,3.15.1:3.16', when='@3.0.0-alpha5')
```

Comment:
Alternatively you could leave the original for versioning, and add an additional requirement for the variants:
```suggestion
      depends_on('petsc+hdf5+mpi')
      depends_on('petsc@3.14.0:3.14,3.15.1:3.15', when='@3.0.0-alpha4')
      depends_on('petsc@3.14.0:3.14,3.15.1:3.16', when='@3.0.0-alpha5')
```